fabric-renderer-indigo/src/client/java/net/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl.java

```diff
@@ -174,8 +174,16 @@ public MutableQuadViewImpl copyFrom(QuadView quad) {
 
 	@Override
 	public final MutableQuadViewImpl fromVanilla(int[] quadData, int startIndex) {
-		System.arraycopy(quadData, startIndex, data, baseIndex + HEADER_STRIDE, QUAD_STRIDE);
+		System.arraycopy(quadData, startIndex, data, baseIndex + HEADER_STRIDE, VANILLA_QUAD_STRIDE);
```

Comment:
Doesn't this just skip the end of the array?

Reply:
`QUAD_STRIDE` and `VANILLA_QUAD_STRIDE` are exactly the same value (this is verified in `EncodingFormat`), but I thought switching it might make this slightly easier to understand.

Reply:
Should also be changed in `toVanilla` then

Reply:
I don't think so. Whether the stride is vanilla should match whether the source array is vanilla, and with this change, it does.

Reply:
Idk, it was confusing for me because I didn't know that the values were equal.